src/main/java/org/jbake/app/Oven.java

```diff
@@ -174,7 +167,7 @@ public void bake() {
                 }
         } finally {
                 db.close();
-//                Orient.instance().shutdown();
```

Comment:
This was removed because it caused an issue while running in JBake's watch mode... but the cause of that may have been fixed since.... could you check if this still breaks watch mode?